Updates/4818_sealed_scroll_case_i.33926.sql

```diff
@@ -0,0 +1,20 @@
+-- fix loot table for i.33926 (Sealed Scroll Case)
+-- https://web.archive.org/web/20090208155218/http://wow.allakhazam.com/db/item.html?witem=33926#Contains_Items
+
+-- remove refloot since we cannot group refloots
+DELETE FROM `item_loot_template` WHERE entry = 33926 AND item = 50604;
+
+-- add all scrolls V/VI individually to Sealed Scroll Case loot table
+INSERT INTO `item_loot_template` (entry, item, ChanceOrQuestChance, groupid, mincountOrRef, maxcount, condition_id, comments) VALUES
```

Comment:
Please add a delete query befor this insert, so it is easier to port/reexeceute.